src/tests/unit/components/NavigationMailbox.vue.spec.js

```diff
@@ -22,8 +22,8 @@
 import {createLocalVue, shallowMount} from '@vue/test-utils'
 import Vuex from 'vuex'
 
-import NavigationMailbox from '../../../components/NavigationMailbox'
-import Nextcloud from '../../../mixins/Nextcloud'
+import NavigationMailbox from '../../../components/NavigationMailbox.js'
```

Comment:
```suggestion
import NavigationMailbox from '../../../components/NavigationMailbox.vue'
```

The linter complained about other tests as well.

Reply:
Works on my machine

Reply:
Oops, I meant the js unit tests.

Does `npm run test:unit` work for you?

Reply:
Nope. I'm checking.